core/src/main/res/values/styles.xml

```diff
@@ -60,6 +60,13 @@
     <item name="cardBackgroundColor">@color/card_view_background</item>
   </style>
 
+  <style name="ThemeOverlay.App.BottomNavigationView" parent="Widget.MaterialComponents.BottomNavigationView">
```

Comment:
It isn't really a ThemeOverlay when you inherit from a parent, this can just follow our widget naming convention